packages/core/src/components/Lightbox/index.tsx

```diff
@@ -140,7 +140,7 @@ export default class Lightbox extends React.PureComponent<LightboxProps, Lightbo
     );
 
     return (
-      <Sheet compact headerShadow noAnimation portal visible header={header} onClose={onClose}>
```

Comment:
instead of the shadow, add a border bottom to the header

Reply:
Switched it to a thick border.